### PR TITLE
Integrate dataman with saas

### PR DESCRIPTION
**Pull request by sedroche**
All the changes needed to provision fh-dataman with fhcap

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,9 +48,9 @@
     "express-bunyan-logger": "1.3.1",
     "fh-cluster": "0.3.0",
     "fh-config": "1.1.0",
-    "fh-db": "feedhenry/fh-db",
+    "fh-db": "~2.0.1",
     "fh-logger": "0.5.1",
-    "fh-mbaas-client": "0.16.5",
+    "fh-mbaas-client": "github:sedroche/fh-mbaas-client#5e2f2df3728fd21a52a5ca6c827d1231a49d5525",
```
Comment (sedroche) [inline package.json]: Update this once version is published and redo shrinkwrap

File: package.json [hunk 2]
```diff
@@ -48,9 +48,9 @@
     "express-bunyan-logger": "1.3.1",
     "fh-cluster": "0.3.0",
     "fh-config": "1.1.0",
-    "fh-db": "feedhenry/fh-db",
+    "fh-db": "~2.0.1",
     "fh-logger": "0.5.1",
-    "fh-mbaas-client": "0.16.5",
+    "fh-mbaas-client": "github:sedroche/fh-mbaas-client#31a4f2f2aa0055e564749822e1c40abb7274cdb3",
```
Comment (sedroche) [inline package.json]: Update this once version is published and redo shrinkwrap

**Review by rachael-oregan — APPROVED** [review]
LGTM

**Comment by rachael-oregan** [comment]
@sedroche additional commits LGTM

File: src/middleware/jwt-authenticate/index.js [hunk 3]
```diff
@@ -17,13 +17,14 @@ export default (options={}) => {
...
   function middleware(req, res, next) {
-    if (!req.headers || !req.headers.authorization) {
-      return next(new UnauthorizedError('Authorisation header has not been set'));
+    const authorization = Object.assign(req.body || {}, req.query || {}, req.headers || {}).authorization;
+    if (!authorization) {
+      return next(new UnauthorizedError('Authorisation has not been set'));
     }
 
-    const [schema='', token=''] = req.headers.authorization.split(' ');
+    const [schema='', token=''] = authorization.split(' ');
```
Comment (pb82) [inline src/middleware/jwt-authenticate/index.js:25]: Will this work as expected (schema and token created under `const`) or does this create two global scoped variables?

Comment (pb82) [reply src/middleware/jwt-authenticate/index.js:25]: Nevermind. I thought destructuring assignments only work for Objects.

**Review by pb82 — APPROVED** [review]
LGTM

**Pull request merged**